helix-core/src/main/java/org/apache/helix/common/caches/CustomizedViewCache.java

```diff
@@ -19,32 +19,24 @@
  * under the License.
  */
 
-import java.util.ArrayList;
 import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
-import java.util.Set;
 
 import org.apache.helix.HelixDataAccessor;
-import org.apache.helix.HelixException;
 import org.apache.helix.PropertyKey;
 import org.apache.helix.PropertyType;
 import org.apache.helix.model.CustomizedView;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.Maps;
 
 /**
  * Cache to hold all CustomizedView of a specific type.
  */
 public class CustomizedViewCache extends AbstractDataCache<CustomizedView> {
   private static final Logger LOG = LoggerFactory.getLogger(CustomizedViewCache.class.getName());
 
-  protected Map<String, CustomizedView> _customizedViewMap;
-  protected Map<String, CustomizedView> _customizedViewCache;
+  private final PropertyCache<CustomizedView> _customizedViewCache;
```

Comment:
Do we still need this CustomizedViewCache file? I thought this would be the same as ExternalViewCache, which is deprecated. And we use PropertyCache<CustomizedView> instead in the place we need it. Please confirm with @jiajunwang .

Reply:
I agree. There should be no extra logic you need to add in addition, once you are using PropertyCache.

Reply:
I chatted with Meng about this. Since both of our PRs has been implemented based on this file, I think it makes more sense to have separate file for customizedViewCache and have it implemented based on PropertyCache as discussed.